samples/dnn/action_recognition.py

```diff
@@ -0,0 +1,174 @@
+import os
+import numpy as np
+import cv2 as cv
+import argparse
+
+parser = argparse.ArgumentParser(description='Use this script to run action classification using 3D ResNet34')
+parser.add_argument('--input', '-i', help='Path to input video file. Skip this argument to capture frames from a camera.')
+parser.add_argument('--model', help='Path to model.')
+parser.add_argument('--classes', default=os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                                '..', 'data', 'dnn', 'action_classification.txt'), help='Path to classes list.')
```

Comment:
Should be just `default=action_classification.txt`. Use `findFile` from `common.py` to get a full path to it.